pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.ejs

```diff
@@ -251,10 +253,15 @@
               <li>Submit your answer to each question with the <strong>Save</strong> button on the question page.</li>
               <li>
                 After you have answered all the questions completely, click here:
-                <button class="btn btn-danger" data-toggle="modal" data-target="#confirmFinishModal">Finish assessment</button>
+                <button class="btn btn-danger" data-toggle="modal" data-target="#confirmFinishModal" <% if (!authz_result.authorized_edit) { %> disabled<% } %>>Finish assessment</button>
```

Comment:
To resolve a conflict as part of the merge 0e0228508d961a74056cb24a13cefeb57788cec4, I deleted the SEB/password cases here to match `master`. This change was introduced in PR #3098 (commit 162a5b921fe4105147366d2f529deef4e4f0b2a5).